File: Android/aryanmishra29_AryanMishra_2125csme1029_2/Patterns/Pattern_5.java
import java.util.*;

public class Pattern_5 {

    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
        int n = scn.nextInt();
        for(int i=0;i<=n/2;i++){
            for(int j=1;j<=(n/2)-i;j++) System.out.print("\t");
            for(int j=1;j<=2*i+1;j++) System.out.print("*\t");
            System.out.println();
        }
        for(int i=n/2-1;i>=0;i--){
            for(int j=1;j<=(n/2)-i;j++) System.out.print("\t");
            for(int j=1;j<=2*i+1;j++) System.out.print("*\t");
            System.out.println();
        }
        // write ur code here

    }
}

File: Web/Vaishnavi-Singh20002_VaishnaviSingh_2125ec1060_02/Patterns/pattern5.cpp
#include <iostream>
using namespace std;

int main(){
    int n;
    cin >> n;
    int a= n/2,b=1;
    for(int i=1;i<=n;i++){
         for (int j = 1; j <= a; j++){
        cout<<"	";
      }
      for (int j = 1; j <= b; j++){
       cout<<"*	";
      }
      if ( i <= n / 2){ a--;
      b+=2;  
      }
        else{
         a++; 
          b-=2;
          }
          cout<<endl; 
          }

    }

    //write your code here
